Abbyy.CloudSdk.V2.Client.Sample/Program.cs

```diff
@@ -67,5 +78,45 @@ private static async Task<List<string>> ProcessImageAsync()
 				return taskInfo.ResultUrls;
 			}
 	    }
+
+	    private static async Task<List<string>> RetryPolicyForSpecificErrorStatusCodeAsync()
```

Comment:
I dont understand naming of this method. What's it do? What verb can explain it? :)

Reply:
code updated